src/neo/Ledger/Blockchain.cs

```diff
@@ -283,15 +283,10 @@ private void OnFillMemoryPool(IEnumerable<Transaction> transactions)
             {
                 if (View.ContainsTransaction(tx.Hash))
                     continue;
-                if (!NativeContract.Policy.CheckPolicy(tx, currentSnapshot))
```

Comment:
Why this was removed?

Reply:
It is checked in `Transaction.Verify()`.

Reply:
![image](https://user-images.githubusercontent.com/3167973/87134606-33056d80-c299-11ea-8f8a-0af0ca111e1f.png)

It's checked the fees but not the blocked accounts, maybe we should unify all of them in only one method

Reply:
The method `CheckPolicy` in PolicyContract is not used, I think that we should move all the logic to this method and do only one call to the native contract

Reply:
Let do it in another PR. This one is to remove the lock.

Reply:
I think that it's better to remove the method `CheckPolicy` the different steps are validated in different areas.

Reply:
Done.